tests/manage/monitoring/conftest.py

```diff
@@ -92,7 +96,7 @@ def measure_stop_ceph_mon(measurement_dir):
     mons = [deployment["metadata"]["name"] for deployment in mon_deployments]
 
     # get monitor deployments to stop, leave even number of monitors
-    split_index = len(mons) // 2 if len(mons) > 3 else 2
+    split_index = split_index if split_index else len(mons) // 2 if len(mons) > 3 else 2
```

Comment:
Could we write this in more readable way instead of creating bit confusing function fixture overloading a variable name?

Reply:
I do want to use the same function when mon quorum is lost/down, i.e 2 mons down out of 3. 
So passing specifically `spilt_index` as 1 here but not able to set `split_index` as a parameter so created fixture to override a value

Reply:
Ah, I see. This makes sense, we just need to make the intentions more clear.

Do I understand it right that we need separate `split_index` fixture for the parametrization to work? In other words, would it work if we set `split_index=None` in definition of `measure_stop_ceph_mon`?

Overall, I would suggest to:

- parametrize the `measure_stop_ceph_mon` fixture via number of mon to go down (instead of split index, which is an internal implementation detail)
- calculate split index based on the above
- if we really need separate `split_index` fixture, it's name would rather be something like `stop_ceph_mon_num`, with some brief explanation in a docstring

Reply:
ack, created `create_mon_quorum_loss` fixture and if it passed as true `split_index` will be set to 1 so that only one mon will be up and running otherwise leave even number of monitors to be up and running.